Blazor/README.md

```diff
@@ -125,12 +123,19 @@ To use, in your project:
     ...    
 ```
   - If using the application template, simply use `appTop` and `appFooter` instead
+  - When calling InstallCDTS in Program.cs, one of the optional parameters is a boolean that needs to be set to true if using the application template
+
+## RefTop/RefFooter
+
+ - By default the user does not need to configure the refTop and refFooter objects for the layout. If required, this can be done in Program.cs through the SetupBase object.
+
+ - When calling the InstallCDTS in Program.cs, it can take an optional SetupBase object which you can use to define the RefTop and RefFooter properties such as ExitScript.
 
 ## Handling CDTS Language Switching
 
 The CDTS and the WET library support both English and French languages but only one can be active at any given time. 
 
-Currently, there is a Blazor component called 'ChangeLang' that takes care of language handling when called. 
+Currently, there is a Blazor component called 'ChangeLang' that takes care of language handling by default.
 
 The component stores and retrieves the page name, changes the language and reloads that same page the user was on in the other language. 
 
```

Comment:
For the "OPTIONAL" sentence, it is good but more details should probably added, probably along with an example.  

In particular, we should explain what the possible values are and what is the impact of each situation.   The following paragraph from the React readme could be adapted:
```
And so, for the `lngLinks` property of the `top` section:
  - If left `undefined` or set to `lngLinks: null` : The CDTS React component will install its default event. This is the recommended option.
  - If set to an empty array (i.e. `lngLinks: []`) : Disables the generation of the standard CDTS language link.
  - If set to any other valid value (e.g. `lngLinks: [{"lang": "fr", "href": "/to_french",	"text": "Français"}]`) : A "normal" language link will be generated accordingly and the CDTS language switch event will NOT be attached. In this case the application will have to call `useCdtsContent().setLanguage(...)` explicitly.
```

Reply:
done